src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arrayOfStyle = sourceString.split(';');
+  const result = {};
+
+  arrayOfStyle.forEach((elem) => {
+    const style = elem.split(':');
+
+    if (style[1]) {
+      result[style[0].replace(/\s+/g, '')] = style[1].trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Although using `forEach` loop is not a mistake, in this task you are encouraged to use array methods like `reduce` to create a new object. This will make your code more readable and concise.